src/makefile.all

```diff
@@ -417,9 +422,15 @@ CFLAGS = $(CFLAGS) -RT-
 CFLAGS  = $(CFLAGS) -DFLEXINT_H -Dflex_int32_t=int -Dflex_int16_t=short
 
 AFLAGS  = -mx -m2 -t -l -DDOSX
-TARGETS = ..\lib\wattcpbw.lib ..\lib\wattcpbw_imp.lib ..\bin\watt-32.dll
-OBJDIR  = build\borland\win32
-MAKEFIL = bcc_w.mak
+
+WATTLIB  = ..\lib\wattcpbw.lib
+WATTILIB = ..\lib\wattcpbw_imp.lib
```

Comment:
Wouldn't it be clearer it these were named:
* `STAT_LIB = ..\lib\wattcpbw.lib`
*  `IMP_LIB = ..\lib\wattcpbw_imp.lib`

In this PR, there is only a single `I` differentiating them.